sdk/identity/identity/test/public/node/authorizationCodeCredential.spec.ts

```diff
@@ -1,136 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import assert from "assert";
-import { AuthorizationCodeCredential } from "../../../src";
-import { TestTracer, SpanGraph } from "@azure/test-utils";
-import { setTracer } from "@azure/core-tracing";
-import { MockAuthHttpClient, assertClientCredentials } from "../../authTestUtils";
-import { setSpan, context as otContext } from "@azure/core-tracing";
-
-describe("AuthorizationCodeCredential", function() {
```

Comment:
This test doesn’t make sense right now since MSAL is doing all of the requests.. This credential is complicated to test to make a recorded test work. I would need to make a web server, go to the web server, record a browser redirect, etc. I think we can get away without having live tests for this credential as long as we do manual tests thoroughly, cc: @KarishmaGhiya help appreciated 🌞

If you (or anyone) know of a simple test I could make that would be useful, beyond just showing “hey we accept parameters”, please let me know!